src/main/scala/CypherExpressable.scala

```diff
@@ -25,4 +26,16 @@ object CypherExpressible {
     def toQuery(double: Double): String =
       double.toString
   }
+
+  implicit object CypherExpressibleBoolean extends CypherExpressible[Boolean] {
+    def toQuery(boolean: Boolean): String =
+      if (boolean) "TRUE" else "FALSE"
+  }
+
+  implicit def optionExpressible[T](implicit serializer: CypherExpressible[T]): CypherExpressible[Option[T]] =
```

Comment:
https://github.com/playframework/playframework/blob/2.3.x/framework/src/play-json/src/main/scala/play/api/libs/json/Reads.scala#L409

looks like they changed this in 2.4.x though, i'll have to look into what their new solution is